cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -530,13 +532,17 @@ def prepare_download(self, file_list, file_name, scratch_dir):
         return tmp_dir, file_name
 
     def resolve_job_url(self):
-        request_par_dic = self.get_request_par_dic()
+        self.par_dic.update(self.get_request_par_dic())
+
+        # what if scw list from request overwrites that in self.par_dict?
+        self.set_scws_related_params(request)
```

Comment:
If that happens, then it is only supposed to overwrite with a different formatting, it's not going to overwrite it with a different one.

Reply:
oh but it's going to change the parameters.
here I added a test to show how it happens:

https://github.com/oda-hub/dispatcher-app/blob/54ed837d68c4e61dd707a7252c8732cdf8bc962b/tests/test_job_management.py#L1318-L1322

I am not even sure it is a bug. A bit of an unintended behavior. Easy to prevent